yt/visualization/particle_plots.py

```diff
@@ -512,7 +512,8 @@ def ParticlePlot(ds, x_field, y_field, z_fields=None, color='b', *args,
         ax_field_template = 'particle_position_%s'
         xf = ax_field_template % ds.coordinates.axis_name[xax]
         yf = ax_field_template % ds.coordinates.axis_name[yax]
-        if (x_field[1], y_field[1]) == (xf, yf):
+        if (x_field[1], y_field[1]) == (xf, yf) \
+                or (x_field[1], y_field[1]) == (yf, xf):
```

Comment:
It's a little awkward with the line continuation character like that. I'd phrase this like:

```
if (x_field[1], y_field[1]) in [(xf, yf), (yf, xf)]:
```

Also it would be great to get a test. You can generate a dataset populated with randomly generated data including some particle fields like this:

```
from yt.testing import fake_random_ds

num_particles = 15
ds = fake_random_ds(16, particles=num_particles)
```

Take a look at `yt/visualization/tests/test_particle_plot.py` for other tests of the particle plot functionality.

Reply:
That's a  good way to write it. Will make the necessary changes.